src/Components/Layout/index.jsx

```diff
@@ -6,6 +6,7 @@ import Admins from '../Admins';
 import Classes from '../Classes';
 import Members from '../Members';
 import Subscriptions from '../Subscriptions';
+import Form from '../Subscriptions/Form';
```

Comment:
You shouldn't use such a generic name like Form, try using SubscriptionForm or SubsForm.